test/fixtures/bn.json

```diff
@@ -0,0 +1,105 @@
+{
+  "valid": [
+    {
+      "dec": "0",
+      "hex": ""
```

Comment:
From a quick look around, it seems the default is to print the `0`.  

https://github.com/node-modules/node-biginteger/
https://github.com/peterolson/BigInteger.js

I'm still not sure what I agree with, but I'll happily change the test based on a semi-consistent consensus.